app/pages/sections/catalogue-solutions/routes.js

```diff
@@ -45,8 +46,12 @@ export const catalogueSolutionsRoutes = (authProvider, addContext, sessionManage
   router.use('/select', catalogueSolutionsSelectRoutes(authProvider, addContext, sessionManager));
 
   router.get('/:orderItemId', authProvider.authorise({ claim: 'ordering' }), withCatch(logger, authProvider, async (req, res) => {
-    const { orderId } = req.params;
+    const { orderId, orderItemId } = req.params;
     const accessToken = extractAccessToken({ req, tokenType: 'access' });
+
+    if (orderItemId !== 'newsolution') {
+      await getOrderItem(orderId, orderItemId, accessToken);
```

Comment:
don't you need the braces? around the params

Reply:
hm oh yeah
although probs could remove the brackets off the method in the controller too as they're already defined and getting passed in

Reply:
I'd keep the braces just to be consistent with the rest